docker/DockerFile

```diff
@@ -4,11 +4,13 @@ COPY --chown=root:root server/ssh /etc/ssh/
 COPY --chown=root:root server/script /opt/sshnet
 COPY --chown=sshnet:sshnet user/sshnet /home/sshnet/.ssh
 
-RUN apk add --no-cache syslog-ng && \
+RUN apk update && apk upgrade --no-cache && \
+    apk add --no-cache syslog-ng && \
     # install and configure sshd
     apk add --no-cache openssh && \
     # install openssh-server-pam to allow for keyboard-interactive authentication
     apk add --no-cache openssh-server-pam && \
+    dos2unix /etc/ssh/* && \
```

Comment:
Without this line, tests did not work. Don't ask how long it took me :)

Reply:
I won't :p

Reply:
I've added comments.

Now I think that it is very difficult to detect and maybe it would be worth adding a check in the library and showing a comment. Of course in other PR.